docs/get-started/alerting.md

```diff
@@ -73,82 +68,64 @@ To use SMTP with a PMM Docker installation:
 
 2. Pass in the `.env` file to Docker run using the `--env-file` flag: ```docker run --env-file=.env -p 443:443 -p 80:80 percona/pmm-server:2```.
 
-3. In the grafana.ini file, add the following section and lines:
-       ` [smtp]`
-       ` enabled = true`
-        `host = name and port of SMTP server (i.e. smtp.gmail.com:465) `
-       ` user = user@domain.com`
-    `    password = app email password` 
-    For information about app passowrds, see the [Gmail documentation](https://support.google.com/mail/answer/185833?hl=en)
-4. Save the file and restart grafana from *(/etc/grafana)* using the following command: `supervisorctl restart grafana`.
+#### Restore SMTP settings following an upgrade
 
+Upgrading PMM will overwrite the **grafana.ini** file with a newer version. If you configured PMM to use SMTP settings via environment variables, you do not need to do anything after an upgrade as your settings will be transferred.
 
-#### Restore SMTP settings following an upgrade
+If the SMTP settings were configured directly in the **grafana.ini** file, make sure to manually back up the .ini file and copy your SMTP settings after the upgrade. This is because upgrading PMM will overwrite the grafana.ini file.
 
-Upgrading PMM will overwrite the **grafana.ini** file with a newer version. If you configured PMM to use SMTP settings via environment variables, you do not need to do anything after an upgrade as your settings will be transferred. If the SMTP settings were configured directly in the **grafana.ini** file, you will need to manually backup the .ini file and copy your SMTP settings after the upgrade. This is because upgrading PMM will overwrite the grafana.ini .
+PMM's root notification policy,**grafana-default-email**, uses Email as the default contact point. To use this default contact, configure it with your email user account information so PMM can out PMM will show a **No Attempts** status next to this contact point.
 
-### Contact points
```

Comment:
This section is incorrect...the steps you outline have nothing to do with notification policies...I believe you are showing how to edit a contact point. Please correct and I would highly recommend a screnshot